scripts/run_kcov.sh

```diff
@@ -54,12 +54,16 @@ for i in target/debug/deps/trust_dns*-* target/debug/deps/*_tests-* ; do
 done
 
 echo "----> ran $test_count test(s)"
-echo "----> merging and uploading to coveralls.io"
 
-if [[ "$test_count" -eq 1 ]] ; then
-  echo kcov --coveralls-id=${TRAVIS_JOB_ID} --report-only target/kcov-* ${last_test}
-  kcov --coveralls-id=${TRAVIS_JOB_ID} --report-only target/kcov-* ${last_test}
-elif [[ "$test_count" -gt 1 ]] ; then
-  echo kcov --coveralls-id=${TRAVIS_JOB_ID} --report-only --merge target/kcov-*
-  kcov --coveralls-id=${TRAVIS_JOB_ID} --report-only --merge target/kcov-*
-fi
\ No newline at end of file
+# echo "----> merging and uploading to coveralls.io"
```

Comment:
codecov.io?

Reply:
Commented out all the coveralls.io stuff see the codecov.io stuff at the bottom.